src/ring-call-channel.c

```diff
@@ -544,6 +553,15 @@ ring_call_channel_class_init(RingCallChannelClass *klass)
       G_PARAM_READABLE | G_PARAM_STATIC_STRINGS));
 
   g_object_class_install_property(
+    object_class, PROP_IMSI,
+    g_param_spec_string("imsi",
+      "IMSI",
+      "Internation Mobile Subscriber Identifer",
```

Comment:
This misspelling is used consistently here, I thought it might be best to preserve it.  If you're certain it's safe, we could fix all instance instead...